doc/release-notes/6467-optimize-permission-lookups-for-a-user.md

```diff
@@ -0,0 +1,9 @@
+The following API have been added:
```

Comment:
Once we've finalized the name of the API endpoint, please add it to the API Guide.